string type
string mapName
string savePath
string pharmacy
string patient
string prescription